LanternMobileTestbed/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <service
+            android:name="org.lantern.mobilesdk.service.LanternService"
+            android:exported="false"
+            android:process=":LanternService"
+            />
```

Comment:
To be able to run the LanternService, it needs to be declared in the app's manifest.  And, it needs to be given a process name to force it to be forked.